Update now-next-legacy-mode.md (#208)
zeit/now-builders: errors/now-next-legacy-mode.md

In order to create the smallest possible lambdas Next.js has to be configured to build for the `serverless` target.

> 1. Serverless Next.js requires Next.js 8 or later, currently this version is out on the `latest` release channel: 

Reviewer: ```suggestion
1. Serverless Next.js requires Next.js 8 or later, to upgrade you can install the `latest` version: 
```
